apps/projects/models.py

```diff
@@ -134,7 +135,10 @@ def create_insight_context(insight: ProjectInsight) -> dict:
     show_ideas = bool(blueprint_types.intersection({"BS", "IC", "MBS", "MIC", "PB"}))
 
     counts = [
-        (_("active participants"), insight.active_participants.count()),
+        (
+            _("active participants"),
+            insight.active_participants.count() + insight.unregistered_participants,
```

Comment:
here active participants are the sum of both registered and unregistered. But when a poll is submitted by an unregistered user, the active participants are not increased, only the unregistered ones. See my comment below in the signal.

Reply:
`active_participants` is a m2m relation to the user model, so we can't increase it for unregistered users, that's why I added the `unregistered_participants` field. And we combine both fields here to reflect the total count of participants (maybe somewhat confusingly still called "active participants").